tests/late_joiners.py

```diff
@@ -198,11 +198,10 @@ def run(args):
                         if args.consensus == "pbft"
                         else args.raft_election_timeout / 1000
                     )
-
                     if node.node_id == cur_primary_id and args.consensus == "pbft":
-                        # if pbft suspend the primary for twice the elecetion timeout
+                        # if pbft suspend the primary for > 2 the elecetion timeout
                         # in order to make sure view changes will be triggered
-                        et += et
+                        et += 1.5 * et
```

Comment:
Minor: Took me a while to parse this. Can we keep the comment in English "suspend the primary for more than twice the election timeout", and make this line `et = et * 2.5` so it more obviously matches the comment? And maybe rename the variable from `et` to `suspend_time` (we modify the suspend time in this case, the election timeout is a constant).